### ISSUE TITLE
ColorControl always hangs after executed action LG C1

### ISSUE THREAD

**Issue by Maxavelli** [Maassoft/ColorControl#378]
So I've now tried 4 different versions of ColorControl and they just hang and ultimately crash after I've power off manually. ColorControl has crashed for me for the last few months and I finally got to testing what's going on. Sometimes in the past it has worked rolling back a version or two but not this time. I am gonna roll back to an even older version later but I wanted to write this.  The tv actually turns on if it's either left to it's on devices letting the screensaver activate where then ColorControl after a few minutes puts the tv in standby or if I press the power off button on my case which I've set to turn off the "display". However after such actions ColorControl hangs. And considering it's been like this for quite some time and I just do know what to do I figured I let you know that there might be an issue  considering ColorControl has worked flawlessly in the past. Also I've block any firmware update urls in my router so I haven't updated my firmware for at least a year so nothing's changed on the tv side at least. Of course I don't know if this is a Windows issue where there just happened that something has broken in my Windows version...I usually update my Windows and drivers as regularly as they are announced which I assume shouldn't be an issue but you never know.

To summarize. Automatic powering on the TV after I manually power it off via the control hasn't worked in months. Likewise if a sleep or wake action is successfully sent to the tv that's the last thing ColorControl does as it just hangs after that. This applies to the Power Off/On Test functionality too as it hangs ColorControl which ultimately crashes after that hang.

I love what you do and appreciate what you've accomplished with this amazing program! I can't imagine what I would do without ColorControl to be honest. And I was perfectly happy with the life extending features for the panel while there's also a bunch of other really cool features that has been introduced over the years that just puts this program in a class of its own!

**Comment by Maassoft** [comment]
Is there something in the log that indicates an error? Something seems to occur that either hangs the event handling or there's an internal exception that stops everything.
Thanks for the kind words!
